components/dashboard/src/data/setup.tsx

```diff
@@ -152,6 +153,7 @@ function initializeMessages() {
         ...Object.values(EnvVarClasses),
         ...Object.values(PrebuildClasses),
         ...Object.values(SCMClasses),
+        ...Object.values(SSHClasses),
```

Comment:
Cache version bump is missing. We can add it a TODO then address before merge, since there're lot of PRs that will change it